### PR TITLE
Reset logs TCP connections every hour

### PR DESCRIPTION
**Pull request by ajacquemot**
### What does this PR do?

Reset logs TCP connections every hour to better balance the traffic over the logs-intake backend, target new nodes when added.

### Motivation

Ease migration and spread.

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: pkg/logs/client/destination.go [hunk 1]
```diff
@@ -45,16 +52,15 @@ type Destination struct {
...
 	once                sync.Once
-	warningCounter      int
 }
 
 // NewDestination returns a new destination.
 func NewDestination(endpoint Endpoint, destinationsContext *DestinationsContext) *Destination {
-	prefix := endpoint.APIKey + string(' ')
 	return &Destination{
-		prefixer:            newPrefixer(prefix),
+		prefixer:            newPrefixer(endpoint.APIKey + " "),
```
Comment (gaetan-deputier) [inline pkg/logs/client/destination.go]: You might want to use your separator const?

**Review by mnshdw — APPROVED** [review]

**Comment by albertvaka** [comment]
Is this good to merge? @ajacquemot

**Comment by KSerrania** [comment]
Is this PR still relevant? If yes, can the conflicts be resolved and the PR merged for the 6.14.0 release?

**Comment by ajacquemot** [comment]
Closing this PR for now as we did now, cc @KSerrania

**Pull request closed without merging**